Source/Events/Processing/BootProcedure.cs

```diff
@@ -85,8 +86,11 @@ void ProcessInParallel()
                 {
                     Parallel.ForEach(_tenants.All, (t) =>
```

Comment:
I don't think this code should know about these. The _tenants.All shouldn't return System and Unknown

Reply:
Makes sense